src/main.sh

```diff
@@ -28,9 +28,9 @@ parseInputs () {
   if [ -n "${INPUT_K8_NAMESPACES}" ]; then
     export k8_namespaces="${INPUT_K8_NAMESPACES}"
   fi
-  helm_secret_chart_version=""
-  if [ -n "${INPUT_HELM_SECRET_CHART_VERSION}" ]; then
-    export helm_secret_chart_version="${INPUT_HELM_SECRET_CHART_VERSION}"
+  helm_create_secret_manager_secret_version=""
+  if [ -n "${INPUT_helm_create_secret_manager_secret_version}" ]; then
```

Comment:
Is this input variable now `${INPUT_helm_create_secret_manager_secret_version}`?

Reply:
i think my find and replace tolowered it.

Reply:
back to upper